postgres/datadog_checks/postgres/settings.py

```diff
@@ -0,0 +1,110 @@
+import psycopg2
+
+from datadog_checks.base import is_affirmative
+from datadog_checks.base.utils.db.utils import DatabaseSetting, DBMAsyncJob
+
+PG_STAT_STATMENTS_MAX_UNKNOWN_VALUE = -1
+TRACK_ACTIVITY_QUERY_SIZE_UNKNOWN_VALUE = -1
+
+
+class PostgresSettings(DBMAsyncJob):
+    """PostgresSettings holds settings queried from the pg_settings table."""
+
+    # Setting names
+    PG_STAT_STATEMENTS_MAX = "pg_stat_statements.max"
+    TRACK_ACTIVITY_QUERY_SIZE = "track_activity_query_size"
+
+    # Queries
+    PG_SETTINGS_QUERY = ("SELECT name, setting FROM pg_settings WHERE name IN ('{}', '{}')").format(
+        PG_STAT_STATEMENTS_MAX, TRACK_ACTIVITY_QUERY_SIZE
```

Comment:
Can any of these settings change dynamically during runtime? I'm wondering if we even need to keep refreshing them.

Reply:
That's a great question. I was curious myself and I checked the docs. Here are the links:
* [pg_stat_statement.max](https://www.postgresql.org/docs/13/pgstatstatements.html)
* [track_activity_query_size](https://www.postgresql.org/docs/13/runtime-config-statistics.html)

Both have the note: 

```
This parameter can only be set at server start.
```